test/e2e/parallel/serviceaccount_test.go

```diff
@@ -20,7 +20,6 @@ func TestDoNotOverrideServiceAccount(t *testing.T) {
 	// given
 	// Skipping the TestDoNotOverrideServiceAccount test instead of deleting it because we will need to create SAs as part
 	// of the environment sub-workspaces so the test & logic will be useful to keep.
```

Comment:
we could drop those now
```suggestion
```

Reply:
removed in https://github.com/codeready-toolchain/toolchain-e2e/pull/768